homeassistant/components/homekit/__init__.py

```diff
@@ -800,18 +800,10 @@ async def async_configure_accessories(self) -> list[State]:
         """Configure accessories for the included states."""
         dev_reg = dr.async_get(self.hass)
         ent_reg = er.async_get(self.hass)
-        device_lookup = ent_reg.async_get_device_class_lookup(
```

Comment:
We can probably drop this function from the registry as only HK uses it

Reply:
Let's drop it.